src/js/lib.js

```diff
@@ -381,21 +367,21 @@ vjs.USER_AGENT = navigator.userAgent;
  * @constant
  * @private
  */
-vjs.IS_IPHONE = (/iPhone/i).test(vjs.USER_AGENT);
-vjs.IS_IPAD = (/iPad/i).test(vjs.USER_AGENT);
-vjs.IS_IPOD = (/iPod/i).test(vjs.USER_AGENT);
-vjs.IS_IOS = vjs.IS_IPHONE || vjs.IS_IPAD || vjs.IS_IPOD;
+var IS_IPHONE = (/iPhone/i).test(USER_AGENT);
+var IS_IPAD = (/iPad/i).test(USER_AGENT);
+var IS_IPOD = (/iPod/i).test(USER_AGENT);
+var IS_IOS = IS_IPHONE || IS_IPAD || IS_IPOD;
```

Comment:
I personally prefer the notion of having all the exports for a file defined in a single place and having it be consistent. It seems a bit like an anti-pattern to have it one way in the entire project, but export at definition for just things related to lib.

I think simply breaking up the files in the first place is going to mean we don't have the giaaant export statement, so perhaps this isn't as big of a deal.